modules/ting_smart_search/ting_smart_search.module

```diff
@@ -19,6 +19,13 @@ function ting_smart_search_menu() {
     'access arguments' => array('configure smart search'),
     'file' => 'includes/ting_smart_search.admin.inc',
   );
+
+  $items['ting/smartsearch/smartsearchrecords'] = array(
+    'title' => 'Ting Smart Search Recors',
```

Comment:
```suggestion
  $items['ting/smart_search/records'] = array(
    'title' => 'Ting smart search records',
```

There is no need to duplicate smart_search in multiple parts of the path. This is one way it could be simplified.

I also suggest making the title casing consistent with the other menu entry.